test/new-e2e/tests/installer/script/default_script_test.go

```diff
@@ -40,6 +41,7 @@ func (s *installScriptDefaultSuite) TestInstall() {
 		"DD_SITE=datadoghq.com",
 		"DD_APM_INSTRUMENTATION_LIBRARIES=java:1,python:2,js:5,dotnet:3",
 		"DD_APM_INSTRUMENTATION_ENABLED=host",
+		"DD_NETWORK_CONFIG_ENABLED=true", // necessary to get process-agent running
```

Comment:
I'm not sure I follow why both this and the removal of units running is needed ? My understanding is that enabling this setting should lead to the previous behavior and require no extra changes

Reply:
This only applies to this specific test

Reply:
This PR changes it so process checks run in core agent by default on Linux, and thus process-agent will not be running. It will start up but then exit because it isn't needed.